zipline/data/bundles/core.py

```diff
@@ -210,9 +208,9 @@ def _make_bundle_core():
     @curry
     def register(name,
                  f,
-                 calendar=trading_days,
-                 opens=open_and_closes['market_open'],
-                 closes=open_and_closes['market_close'],
+                 calendar=nyse_cal,
```

Comment:
I think that users should still have the option to pass the actual calendar though